src/test/java/codeu/controller/AdminServletTest.java

```diff
@@ -60,7 +66,16 @@ public void testDoGet_UserIsAdmin() throws IOException, ServletException {
 
         adminServlet.doGet(mockRequest, mockResponse);
 
-        Mockito.verify(mockSession).setAttribute("admin_message", "Hello " + admin + "! Welcome to the admin page!");
+        Mockito.verify(mockSession).setAttribute("adminMessage", "Hello " + admin + "! Welcome to the admin page!");
+
+        int numUsers = UserStore.getInstance().getNumUsers();
```

Comment:
For your tests it would be better if you set the values for your stores (maybe insert some fake users and messages or mock the stores themselves) and verify that they are what you set them to, because otherwise you may get the value zero for a number of reasons

Reply:
I'm trying to mock the stores and I'm having trouble;
![image](https://user-images.githubusercontent.com/11599574/40569393-7e9b234e-603e-11e8-8622-dd9888793136.png)

When I do this, I do get the value zero returned in my tests like you mentioned, so the test fails. I was thinking that this was because in my servlet I do ConversationStore.getInstance().getNumConversations(), so Mockito.when isn't properly making that line return 123 in my tests because getInstance() is there. But I can't exactly mock getInstance() either since it's a static method. Am I on the wrong track here?